src/core/src/package_managers/YumPackageManager.py

```diff
@@ -863,42 +853,36 @@ def try_mitigate_issues_if_any(self, command, code, out):
             if issue_mitigated:
-                self.composite_logger.log_debug('\nPost mitigation, invoking package manager again using: ' + command)
+                self.composite_logger.log_debug('Post mitigation, invoking package manager again using: ' + command)
                 code_after_fix_attempt, out_after_fix_attempt = self.env_layer.run_command_output(command, False, False)
                 return self.try_mitigate_issues_if_any(command, code_after_fix_attempt, out_after_fix_attempt)
         return code, out
 
     def check_known_issues_and_attempt_fix(self, output):
         """ Checks if issue falls into known issues and attempts to mitigate """
-        self.composite_logger.log_debug("Output from package manager containing error: \n|\t" + "\n|\t".join(output.splitlines()))
-        self.composite_logger.log_debug("\nChecking if this is a known error...")
+        self.composite_logger.log_debug("[YPM] Checking against known errors: [Out={0}]".format(output))
         for error in self.known_errors_and_fixes:
             if error in output:
-                self.composite_logger.log_debug("\nFound a match within known errors list, attempting a fix...")
+                self.composite_logger.log_debug("[YPM] Found a match within known errors list, attempting a fix...")
                 self.known_errors_and_fixes[error]()
                 return True
 
-        self.composite_logger.log_debug("\nThis is not a known error for the extension and will require manual intervention")
+        self.composite_logger.log_error("[YPM] Customer Environment Error: Not a known error. Please investigate and address. [Out={0}]".format(output))
         return False
 
     def fix_ssl_certificate_issue(self):
         command = self.yum_update_client_package
-        self.composite_logger.log_debug("\nUpdating client package to avoid errors from older certificates using command: [Command={0}]".format(str(command)))
+        self.composite_logger.log_debug("[Customer-environment-error] Updating client package to avoid errors from older certificates using command: [Command={0}]".format(str(command)))
         code, out = self.env_layer.run_command_output(command, False, False)
         if code != self.yum_exitcode_no_applicable_packages:
-            self.composite_logger.log('[ERROR] Package manager was invoked using: ' + command)
-            self.composite_logger.log_warning(" - Return code from package manager: " + str(code))
-            self.composite_logger.log_warning(" - Output from package manager: \n|\t" + "\n|\t".join(out.splitlines()))
-            self.telemetry_writer.write_execution_error(command, code, out)
-            error_msg = 'Unexpected return code (' + str(code) + ') from package manager on command: ' + command
+            error_msg = 'Customer environment error (expired SSL certs):  [Command={0}][Code={1}][Out={2}]'.format(command,str(code),out)
```

Comment:
nit: we should have str(out) in format(...) incase out has any special characters

Reply:
I realized we don't want this addition to go into status, only into the logs.
Moved out into just the error logging.
str(x) is only required if there's a risk the type of input is not str. Many places here unnecessarily have it but it's forcing type safety (good thing).